ssl/t1_trce.c

```diff
@@ -13,6 +13,10 @@
 
 /* Packet trace support for OpenSSL */
 #include "internal/nelem.h"
+#include "internal/ssl_unwrap.h"
+#include "internal/quic_predef.h"
+#include "internal/quic_trace.h"
+#include "quic/quic_local.h"
```

Comment:
I think we don't need quic_local.h
removing those two headers does not break compilation at least in my local workspace.